Packs/SymantecCloudSOC/Integrations/SymantecCloudSOC/SymantecCloudSOC.py

```diff
@@ -0,0 +1,489 @@
+        CommandResults: A CommandResults object that contains the events in a table format.
+    """
+
+    events: list[dict] = []
+    investigate_logs_events: list = []
+    incident_logs_events: list = []
+    hr = ""
+    demisto.debug(f'SymantecEventCollector: last_run is {last_run} and the command is {demisto.command()}')
+    investigate_logs_events, incident_logs_events, _ = get_all_events(client, first_fetch_time,
+                                                                      first_fetch_time_investigate,
+                                                                      last_run, limit)
+    if investigate_logs_events:
+        events.extend(investigate_logs_events)
+        hr += tableToMarkdown(
+            name="Investigate Logs Events",
+            t=investigate_logs_events,
+            headerTransform=string_to_table_header,
+            headers=HEADERS.get("Investigate_logs"),
+        )
+    else:
+        hr += "No events found for investigate logs.\n"
+    if incident_logs_events:
+        events.extend(incident_logs_events)
+        hr += tableToMarkdown(
+            name="Incident Logs Events",
+            t=incident_logs_events,
+            headerTransform=string_to_table_header,
+            headers=HEADERS.get("Incident_logs"),
+        )
+    else:
+        hr += "No events found for incident logs.\n"
+
+    return events, CommandResults(readable_output=hr, raw_response=events)
+
+
+def add_fields_to_event(event: dict, log_type: str) -> None:
+    """
+    Adds the _time and type keys to the events.
+    Args:
+        events (List[Dict]): A list of events to add the _time and type keys to.
+    Returns:
+        list: The events with the _time and type keys.
+    """
+    if event:
+        if log_type == "Incident_logs":
+            event['_time'] = event.get('incident_start_time')
+            event['type'] = "Detect incident"
+        elif log_type == "Investigate_logs":
+            event['_time'] = event.get('created_timestamp')
+            event['type'] = "Investigate"
+
+
+def dedup_by_id(last_run: dict, events: list, log_type: str, limit: int,
+                number_of_events: int, last_fetch: str) -> Tuple[list, dict]:
+    """
+    Dedup mechanism for the fetch to check both log_id and created_timestamp/incident_start_time
+    (since timestamp can be duplicate)
+    Args:
+        last_run (dict): Last run.
+        events (list): List of the events from the API.
+        log_type (str): the log type.
+        limit (int): The number of events to return.
+        number_of_events (int): The number of event we already fetched
+        last_fetch (str): Last fetch time.
+    Returns:
+        - list of events to send to XSIAM.
+        - The new last_run (dictionary with the relevant timestamps and the events ids).
+        - The new last_run timestamps.
+    """
+    last_run_ids = dict_safe_get(last_run, [f'{log_type}-ids'], default_return_value=[])
```

Comment:
Why do you need `default_return_value`?

Reply:
if we don't have the ids list in the first run we will get None, but we want it as a list in order to use append on the list.